web/src/lightbox.ts

```diff
@@ -671,7 +671,14 @@ export function initialize(): void {
             return;
         }
 
-        open_image($original_media_element);
+        // Because a user could conceivably copy and paste a media reference
+        // from another message, and because we are selecting the original
+        // element above based on a data-url value that is therefore not
+        // guaranteed to be unique, we pass the first (possibly only) media
+        // element returned. The logic below for removing and adding the
+        // "selected" class ensures that the correct thumbnail will
+        // still be highlighted.
+        open_image($original_media_element.first());
```

Comment:
Would it be better to do the `.first()` above, where `$original_media_element` gets defined? We'd need to tweak the length check, but that would make the singular name feel appropriate; as it is, I could easily see someone adding code later in this function assuming `$original_media_element` is a singular element.